rero_ils/modules/documents/serializers/base.py

```diff
@@ -0,0 +1,268 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2022 RERO
+# Copyright (C) 2022 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Mixin helper class."""
+
+from abc import ABC, abstractmethod
+
+from rero_ils.modules.contributions.utils import \
+    get_contribution_localized_value
+
+CREATOR_ROLES = [
+    'aut', 'cmp', 'cre', 'dub', 'pht', 'ape', 'aqt', 'arc', 'art', 'aus',
+    'chr', 'cll', 'com', 'drt', 'dsr', 'enj', 'fmk', 'inv', 'ive', 'ivr',
+    'lbt', 'lsa', 'lyr', 'pra', 'prg', 'rsp', 'scl'
+]
+
+
+class BaseDocumentFormatterMixin(ABC):
+    """Base formatter class."""
+
+    def __init__(self, record):
+        """Initialize RIS formatter with the specific record."""
+        self.record = record
+
+    @abstractmethod
+    def format(self):
+        """Return RIS export for single record."""
```

Comment:
Is it really RIS specific or it a generic class?

Reply:
Nop, I have updated the docstring